### PR TITLE
Feature/idva5 158 company number

### PR DESCRIPTION
**Pull request by hemerem-ch**
- [ ] Create interface
- [ ] Added endpoint
- [ ] Add validation for company format
- [ ] Front error added
- [ ] Backend and backend connection with Axios
- [ ] Added page re-direct
- [ ] Session to save company value added
- [ ] Test for controller, service and validation added

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by hemerem-ch** [comment]
The test below work in the local env only if the backend is running, if the backend is not running the test fail. Also the test trigger the pipeline to fail, not sure if it is due to the dependency to the backend

/* test("POST" + LIMITED_COMPANY_NUMBER, async () => {
    const res = await router
        .post(BASE_URL + LIMITED_COMPANY_NUMBER)
        .send({ companyNumber: "NI038379" });
    expect(res.status).toBe(302);
}, 1000);// Redirect status code - valid company number */

File: src/test/src/controllers/companyLookupControllerRouter.test.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import supertest from "supertest";
```
Comment (swalke-ch) [inline src/test/src/controllers/companyLookupControllerRouter.test.ts:1]: need to rename the file correctly. Router not needed in name

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by swalke-ch — APPROVED** [review]

**Review by atedros-ch — APPROVED** [review]
LGTM

**Pull request merged**